gh_perf_review.py

```diff
@@ -96,8 +96,8 @@ def _req(url: str, **kwargs: Any) -> Response:
     return Response(json.load(resp), _parse_link(resp.headers['link']))
 
 
-def _get_all(url: str, **kwargs: Any) -> list[dict[str, Any]]:
-    url = f'https://api.github.com{url}'
+def _get_all(api_root: str, url: str, **kwargs: Any) -> list[dict[str, Any]]:
+    url = f'{api_root}{url}'
```

Comment:
can probably move this out to main -- no idea why the url gets formatted here but not in the other 🤔